### ISSUE TITLE
armv7 docker image crashes on startup for v1.15.0 and v1.15.1

### ISSUE THREAD

**Issue by petertrr** [dungeon-revealer/dungeon-revealer#1394]
I'm running dungeon-revealer on Raspberry Pi 3B+:
```bash
$ uname -m
armv7l
$ docker --version
Docker version 20.10.7, build f0df350
```

Dungeon-revealer crashes on startup when run from images v1.15.0 and v1.15.1:

```bash
$ docker run --rm dungeonrevealer/dungeon-revealer:v1.15.0


#
# Fatal error in , line 0
# unreachable code
#
#
#
#FailureMessage Object: 0x7eaf65fc
```
v1.14.1 works fine:
```bash
$ docker run --rm dungeonrevealer/dungeon-revealer:v1.14.1

Starting dungeon-revealer.

Configuration:
- HOST: 0.0.0.0 
- PORT: 3000
- PUBLIC_URL: <none>

dungeon-revealer is reachable via the following addresses:

- http://127.0.0.1:3000
- http://172.17.0.2:3000

Player Section: http://127.0.0.1:3000
DM Section: http://127.0.0.1:3000/dm

-------------------

```

I'm not very familiar with node, so don't know, how to debug it further. I'll be happy to provide additional info, if needed.

**Comment by maxb2** [comment]
It seems that alpine3.13 is using an updated version of musl that uses time64 (a 64-bit library) that will not run on 32-bit hardware (raspi). The quickest fix is to downgrade the base docker image to alpine3.12. See #1397. I want to test it on real hardware before it gets merged. In the long term we need to find another base image such as [node:16-slim](https://hub.docker.com/_/node/?tab=tags&page=1&ordering=last_updated&name=16-slim) which is based on Debian buster.

- Koenkk/zigbee2mqtt#7662
- nodejs/docker-node#1477
- [Release_Notes_for_Alpine_3.13.0](https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.13.0)

**Comment by maxb2** [comment]
@petertrr you can use `dungeonrevealer/dungeon-revealer:dev` until `v1.15.2` gets released.

**Comment by petertrr** [comment]
> @petertrr you can use `dungeonrevealer/dungeon-revealer:dev` until `v1.15.2` gets released.

@maxb2 wow, thanks for the quick fix!

**Comment by n1ru4l** [comment]
@petertrr Now published as v1.16.0 🎉  https://github.com/dungeon-revealer/dungeon-revealer/releases/tag/v1.16.0
